usaspending_api/awards/serializers.py

```diff
@@ -22,11 +22,27 @@ class Meta:
         fields = '__all__'
 
 
+class ProcurementSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Procurement
+        fields = '__all__'
+
+
+class FinancialAssistanceAwardSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = FinancialAssistanceAward
+        fields = '__all__'
+
+
 class AwardSerializer(serializers.ModelSerializer):
 
     recipient = LegalEntitySerializer(read_only=True)
     awarding_agency = AgencySerializer(read_only=True)
     funding_agency = AgencySerializer(read_only=True)
+    procurement_set = ProcurementSerializer(many=True, read_only=True)
```

Comment:
If it is empty it will be an empty list, but I can add a check to only drop it in if it exists.